NewDawn/NewDawn/ViewControllers/ChatPageViewControllers/ChatRoomViewController.swift

```diff
@@ -302,50 +327,61 @@ class ChatRoomViewController: MessagesViewController {
     }
     
     @IBAction func imTakenButtonTapped(_ sender: Any) {
-        fetchEndUserProfile() {
-            profile in
-            if profile.takenBy == -1 {
```

Comment:
I’ll have a try. I put here cuz if I put it in the later part, it will be displayed very slowly, but here it will be displayed before message cells come out.